src/builder/arg.rs

```diff
@@ -1289,11 +1289,16 @@ impl<'help> Arg<'help> {
     /// `.min_values(2)`, and this argument would be satisfied if the user provided, 2 or more
     /// values.
     ///
-    /// **NOTE:** This does not implicitly set [`Arg::multiple_occurrences(true)`]. This is because
+    /// **NOTES:**
+    /// * This does not implicitly set [`Arg::multiple_occurrences(true)`]. This is because
     /// `-o val -o val` is multiple occurrences but a single value and `-o val1 val2` is a single
```

Comment:
Style: We generally have distinct `**NOTE:**`s rather than a bulleted list

If we do keep this, I'd prefer the continued text to be indented